_posts/2022-11-25-ultrastar-on-mobile-with-ultrastar-play.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "UltraStar Play: New Game Instance Supports Mobile Devices"
+date:   2022-11-25 22:12:00 +0200
+categories: news
+---
+Ever wanted to play UltraStar on your smartphone? Look no further!
+
+__UltraStar Play__ is a redevelopment of the UltraStar game using the Unity engine. This allows to build it not only for desktop platforms but also for Android and iOS.
+
+The game is already fun to play, although not yet as feature complete as UltraStar Deluxe.
+
+On the other hand, UltraStar Play offers some unique features, for example easy smartphone connection and a rich song editor with drag'n'drop editing.
```

Comment:
`On the other hand` is missing a "on one hand`. Maybe replace.

"comes with a companion app so you don't need to bring extra microphones but instead can use your phones"